app/lib/postcodeLinks.js

```diff
@@ -31,6 +31,14 @@ function addUrl(postcodeLocationDetails, inputList) {
   });
 }
 
+function addExternalUrl(postcodeLocationDetails, location) {
+  if (location && postcodeLocationDetails && postcodeLocationDetails.location) {
+    return `https://www.nhs.uk/service-search/Chlamydia-free-online-tests-for-u-25s/${location}/Results/105/${postcodeLocationDetails.location.lon}/${postcodeLocationDetails.location.lat}/344/0?distance=25`;
+  }
+  return false;
```

Comment:
There is no need to return false and in fact, returning different types from the same function is likely to cause confusion in the future.

Reply:
Kind of moved from `undefined` to `false` which are both falsey. Moving back to undefined then.